homeassistant/components/sensor/rtorrent.py

```diff
@@ -33,7 +28,7 @@
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_URL): cv.url,
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-    vol.Optional(CONF_MONITORED_VARIABLES, default=ALL_SENSOR_TYPES): vol.All(
+    vol.Optional(CONF_MONITORED_VARIABLES, default=SENSOR_TYPES): vol.All(
```

Comment:
This should be a list of strings not a list with one dict. Change to:
```py
default=list(SENSOR_TYPES)
```

Reply:
Sorry I totally missed that 🙈